tests/assertions/native-functions.n

```diff
@@ -132,29 +132,38 @@ if let <yes module> = intoModule(imp "./imports/unit-test.n") {
 
 	assert value len(results) == 4
 
-	assert value results[0] == {
+	assert value (results |> itemAt(0)) == yes({
 		hasPassed: true
 		fileLine: 3
 		unitTestType: "value"
 		possibleTypes: none
-	}
+	})
 
-	assert value results[1] == {
+	assert value (results |> itemAt(1)) == yes({
 		hasPassed: false
 		fileLine: 5
 		unitTestType: "value"
 		possibleTypes: none
+	})
+
+	for (result in (results |> subsection(2, 4))) {
+		assert value result.hasPassed
```

Comment:
Does this not fail for the fourth test? The fourth result intentionally does not pass

Reply:
It's inclusive and then exclusive, like these functions in other languages.